install/systemd.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: docs
+title: Run Airsonic with Systemd
+permalink: /docs/install/systemd/
+---
+##### Systemd
+
+Systemd is an init system for most linux systems. It allows one to run
+airsonic on boot.
+
+1. Setup dedicated airsonic user
+
+```
+useradd airsonic
```

Comment:
Why not go a bit further and do something like : 
`adduser --system --shell /sbin/nologin --no-create-home airsonic`